ADAL/src/request/ADAuthenticationRequest+WebRequest.m

```diff
@@ -89,7 +130,13 @@ - (void)requestCode:(MSIDAuthorizationCodeCallback)completionBlock
     MSID_LOG_VERBOSE(_requestParams, @"Requesting authorization code");
     MSID_LOG_VERBOSE_PII(_requestParams, @"Requesting authorization code for resource: %@", _requestParams.resource);
 
-    [ADWebAuthController startWithRequest:_requestParams promptBehavior:_promptBehavior context:_context completion:^(MSIDWebviewResponse *response, NSError *error) {
+    NSString *refreshToken = nil;
+    if (_promptBehavior != AD_FORCE_PROMPT && _promptBehavior != AD_PROMPT_ALWAYS && [_context useRefreshTokenForWebview])
```

Comment:
perhaps this can be simplified to _promptBehavior == AD_PROMPT_AUTO, as refresh session is same as prompt always in most cases.

Reply:
I will use your suggestion here, although the documentation for refresh session says it does not prompt in some cases where cookies are available. We don't use AD_PROMPT_REFRESH_SESSION input.